public_html/wp-content/plugins/pattern-directory/includes/stats.php

```
@@ -58,85 +58,71 @@ function get_meta_field_schema() {
 			'count-patterns'                 => array(
 				'description' => __( 'The total number of pattern posts.', 'wporg-patterns' ),
 				'type'        => 'integer',
-				'default'     => 0,
```

Why remove the defaults?

---

If the stat schema changes at some point, and there are new stats that we didn't capture in previous snapshots, having defaults could be misleading. When we export a snapshot that's missing a field from the schema, and that field has a defined default, it will appear that the snapshot just has the default value. But if no default is defined, that snapshot will just have an empty value for that field.

---

Ah, that makes sense. thanks for explaining 👍🏻